cmd/istioctl/istioctl_test.go

```diff
@@ -15,159 +15,359 @@
 package main
 
 import (
+	"errors"
+	"fmt"
 	"testing"
 
-	"istio.io/manager/test/mock"
+	"strings"
+
+	"istio.io/manager/apiserver"
+	"istio.io/manager/model"
 )
 
-func rootSetup(t *testing.T) {
-	config = mock.MakeRegistry()
+type StubClient struct {
```

Comment:
thought: we have `test/mock/config.go` for mocking the config registry. if it makes it easier might be able to wrap the client around it. what you have also works.